backup-index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quizzzy</title>
+    <link rel="stylesheet" href="styles/index.css" />
+    <link href="src/fontawesome/css/all.css" rel="stylesheet" />
+  </head>
+  <body class="app-grid">
+    <header class="app-grid__header centered-grid">
+      <div class="header__heading-box centered-grid">
+        <h1 class="header__heading">Quizzy</h1>
+      </div>
+    </header>
+
+    <main class="app-grid__content centered-grid">
+      <section class="quizcard__card">
+        <div class="quizcard__bookmark--box">
```

Comment:
Again, it seems like a lot of `div`-elements here as well. Please check whether you need them or if you can apply the styles to the icons directly.
Also, `--something` in BEM is considered a modification of an existing style, this however is a element within your block. Or a entirely new block if you really need it:
```suggestion
        <div class="bookmark-box">
```